cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -629,12 +630,10 @@ def configure_kubelet(dns, ingress_ip):
     kubelet_opts['allow-privileged'] = 'true' if privileged else 'false'
 
     if is_state('kubernetes-worker.gpu.enabled'):
-        if get_version('kubelet') < (1, 6):
-            hookenv.log('Adding --experimental-nvidia-gpus=1 to kubelet')
-            kubelet_opts['experimental-nvidia-gpus'] = '1'
-        else:
-            hookenv.log('Adding --feature-gates=Accelerators=true to kubelet')
-            kubelet_opts['feature-gates'] = 'Accelerators=true'
+        hookenv.log('Adding '
+                    '--feature-gates=Accelerators=true,DevicePlugins=true '
+                    'to kubelet')
+        kubelet_opts['feature-gates'] = 'Accelerators=true,DevicePlugins=true'
```

Comment:
Will this work with 1.7? According to the 1.7 kubelet docs here, DevicePlugins doesn't exist yet: https://v1-7.docs.kubernetes.io/docs/admin/kubelet/

Checking with @tvansteenburgh to see if we still need to support 1.7

Reply:
Oh, nevermind, I see now that we only enable GPU support on 1.9+.

Reply:
We do not install cuda anymore so any version less than 1.9 will not have gpu support.